crates/routes/src/post_link_tags.rs

```diff
@@ -0,0 +1,29 @@
+use actix_web::{error::ErrorBadRequest, web::Query, *};
+use anyhow::anyhow;
+use lemmy_utils::{request::fetch_post_link_tags, LemmyError};
+use lemmy_websocket::LemmyContext;
+use serde::Deserialize;
+use url::Url;
+
+#[derive(Deserialize)]
+struct Params {
+  url: String,
+}
+
+pub fn config(cfg: &mut web::ServiceConfig) {
+  cfg.route("/post_link_tags", web::get().to(get_post_links_response));
```

Comment:
This should be under `/api/v3/` just like other endpoints. Otherwise its just confusing.